src/components/HeaderBar/HeaderBar.js

```diff
@@ -1,18 +1,18 @@
-import React from "react";
-import { View, Text, Image } from "react-native";
-import styles from "./styles";
+import React from 'react';
```

Comment:
Use double quotes instead of single

Reply:
Sure! I'll change the linting rules and set Double quotes instead of single quotes